stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -113,6 +114,14 @@ public fun DefaultMessageItem(
     onLongItemClick: (Message) -> Unit = {},
     onThreadClick: (Message) -> Unit = {},
     onImagePreviewResult: (ImagePreviewResult?) -> Unit = {},
```

Comment:
Probably not the right time, as there is a separate task to introduce date and thread separator slot APIs.

Reply:
Yeah, good call - we can do that in a separate task to have everything done in one place! :]